packages/nextjs/types/abitype/abi.d.ts

```diff
@@ -7,15 +7,3 @@ declare module "abitype" {
     AddressType: AddressType;
   }
 }
-
-declare module "viem/node_modules/abitype" {
```

Comment:
removed this since it worked with different abitype versions (1.0.0 in viem, 1.0.5 in our package.json)